README.md

```diff
@@ -21,6 +21,17 @@ Whether you are on VS Code Stable or VS Code Insiders, if you would like to work
 
 Please follow the [Python Extension ReadMe](https://github.com/microsoft/vscode-python/blob/main/README.md) instructions to get started and visit the [Python Documentation](https://code.visualstudio.com/docs/python/jupyter-support) to learn more about how the Python and Jupyter Extension are working together to provide an optimum Python notebooks experience.
 
+### Run by Line
+
+To start a lightweight debugging session and run code cells line by line in Python notebooks, press `F10` while selecting a cell or click the Run by Line button on the cell toolbar.
+
+Once you start a Run by Line session the Variable Explorer will appear and variable values will update as you iterate through your code.
+
+To run through the rest of the cell during a Run by Line session hit `Ctrl+Enter`. To stop, you can click the interrupt button on the left side of the cell.
+
+To set it up, follow the steps [here](https://github.com/microsoft/vscode-jupyter/wiki/Setting-Up-Run-by-Line-and-Debugging-for-Notebooks).
+<img src=https://raw.githubusercontent.com/microsoft/vscode-jupyter/main/images/runbyline.gif?>
```

Comment:
Looks like this GIF needs to be updated to the new UI.

Reply:
Some of the wording is a bit off to me. Might be worth looping in @claudiaregio ?

Reply:
I did update the gif, its on this PR.

Also that wording is from Claudia :v
But I'll take suggestions, anything you'd like to change?

Reply:
's all good then